ts/components/VersionInfoOverlay.tsx

```diff
@@ -47,14 +47,10 @@ const styles = StyleSheet.create({
 
 const VersionInfoOverlay: React.FunctionComponent<Props> = (props: Props) => {
   const appVersion = getAppVersion();
-  const serverInfo = props.serverInfo;
-  const serverVersion = serverInfo ? serverInfo.version : "?";
 
   return (
     <View style={styles.versionContainer} pointerEvents="box-none">
-      <Text style={styles.versionText}>
-        {`app: ${appVersion}`} - {`backend: ${serverVersion}`}
-      </Text>
+      <Text style={styles.versionText}>{`app: ${appVersion}`}</Text>
```

Comment:
```suggestion
      <Text style={styles.versionText}>{`v${appVersion}`}</Text>
```
Since we have only one version to show, could it be a good idea?

Reply:
Sure, changed in ab75551108e508f2a3daa2a7d2838356490dd66c